code/game/machinery/doors/windowdoor.dm

```diff
@@ -105,12 +91,12 @@
 		return 1
 
 /obj/machinery/door/window/open()
-	if (src.operating == 1) //doors can still open when emag-disabled
+	if (operating == TRUE) //doors can still open when emag-disabled
 		return 0
 	if (!ROUND_IS_STARTED)
 		return 0
-	if(!src.operating) //in case of emag
-		src.operating = 1
+	if(!operating) //in case of emag
```

Comment:
This check is unnecessary, if we get to this point we know operating is FALSE already.